doc-content/enterprise-only/release-notes/rn-771-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,72 @@
+[id='rn-771-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.7.1
+
+{PRODUCT} 7.7.1 provides increased stability and fixed issues listed in this section.
+
+ifdef::PAM[]
+[NOTE]
+====
+{PRODUCT_DM} fixed issues apply to {PRODUCT_PAM} as well. For a list of {PRODUCT_DM} 7.7.1 fixed issues, see the https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.7/html-single/release_notes_for_red_hat_decision_manager_7.7/index[_Release Notes for Red Hat Decision Manager 7.7_]
+====
+
+== {CENTRAL}
+
+* In {CENTRAL}, `NotificationListener` is starting without ISO expression [https://issues.redhat.com/browse/RHDM-1278[RHDM-1278]]
+* Stored credentials are not being encrypted when a user import a git repository [https://issues.redhat.com/browse/RHPAM-2838[RHPAM-2838]]
+* {CENTRAL} latency is correlated with the number of group memberships [https://issues.redhat.com/browse/RHPAM-2708[RHPAM-2708]]
+* In the test scenario designer, wrong test results are shown if the rules are not fired [https://issues.redhat.com/browse/RHDM-1330[RHDM-1330]]
+
+endif::[]
+
+ifdef::DM[]
+
+== {CENTRAL}
+* An exception is thrown when a test scenario containing a tag is renamed without saving [https://issues.redhat.com/browse/RHPAM-2674[RHPAM-2674]]
+* In guided rule editor, `java.time.format.DateTimeFormatter` is added multiple times to the rules when you use the use `LocalDate` type [https://issues.redhat.com/browse/RHDM-1318[RHDM-1318]]
+* In the *Data* tab of Guided Rule Templates, enumerated values are not displayed in the multiple select drop-down [https://issues.redhat.com/browse/RHDM-1219[RHDM-1219]]
+
+endif::[]
+
+ifdef::PAM[]
+
+== Process engine
+
+* When an intermediate timer from the process is triggered, the event sub-process with the timer is fired [https://issues.redhat.com/browse/RHPAM-2864[RHPAM-2864]]
+* An error boundary event is unable to handle an exception thrown by reusable sub-process node [https://issues.redhat.com/browse/RHPAM-2782[RHPAM-2782]]
+* The test that starts with high number of processes along with the high timers, the server throws an `OutOfMemoryError` error before all the process instances are being created [https://issues.redhat.com/browse/RHPAM-2912[RHPAM-2912]]
```

Comment:
Suggested rewrite:
When a test starts with a high number of processes and timers, then the server throws an OutOfMemoryError error before all the process instances are created

Reply:
Done